File: 1537-maximum-score-after-splitting-a-string/maximum-score-after-splitting-a-string.cpp
class Solution {
public:
    int maxScore(string s) {
        int max_count=0;
        for (int i=0;i<s.length()-1;++i){
            int left_count=count(s.begin(),s.begin()+i+1,'0');
            int right_count=count(s.begin()+1+i,s.end(),'1');
            max_count=max(max_count,left_count+right_count);
        }
        return max_count;
    }
};

File: 1537-maximum-score-after-splitting-a-string/maximum-score-after-splitting-a-string.py
class Solution:
    def maxScore(self, s: str) -> int:
        max_count=0
        
        
        for i in range(len(s)-1):
            left=right=''
            left_substr=s[:i+1]
            right_substr=s[i+1:]
            left_count=left_substr.count('0')
            right_count=right_substr.count('1')
            max_count=max(max_count,right_count+left_count)
        return max_count

